src/main/java/com/pixonic/ctop/metrics/CurrentMetrics.java

```diff
@@ -13,13 +13,15 @@
     private static final String KEY_PROPERTY = "scope";
     private static final String ATTRIBUTE = "Count";
 
-    CurrentMetrics(long interval, MBeanServerConnection remote, String keySpace) {
-        super(interval, remote, keySpace);
+    CurrentMetrics(long interval, MBeanServerConnection remote, String keySpace, TargetType targetType, MetricsType metricsType, MetricsCollector metricsCollector) {
+        super(interval, remote, keySpace, targetType, metricsType, metricsCollector);
     }
 
-    @Override public void printMetrics() throws Exception {
-        ObjectName readObjectName = new ObjectName("org.apache.cassandra.metrics:type=Table,keyspace=" + keySpace + ",scope=*,name=ReadLatency");
-        ObjectName writeObjectName = new ObjectName("org.apache.cassandra.metrics:type=Table,keyspace=" + keySpace + ",scope=*,name=WriteLatency");
+    @Override
+    public void printMetrics() throws Exception {
+        String ksValue = targetType.equals(TargetType.ALL) ? "*" : keySpace;
```

Comment:
Potential NPE

Reply:
Changed to 

`String ksValue = TargetType.ALL.equals(targetType) ? "*" : keySpace;`